lepton/image.go

```diff
@@ -564,9 +564,9 @@ func DownloadFileWithProgress(filepath string, url string, timeout int) error {
 }
 
 // DownloadFile downloads file using URL
-func DownloadFile(filepath string, url string, timeout int, showProgress bool) error {
-	log.Info("Downloading..", url, "to", filepath)
-	out, err := os.Create(filepath + ".tmp")
+func DownloadFile(fpath string, url string, timeout int, showProgress bool) error {
+	log.Info("Downloading..", url, "to", fpath)
```

Comment:
i'd remove this but everything else looks fine

Reply:
done, we can merge this PR now @eyberg